tests/BlazorServerCallsGraph/appsettings.json

```diff
@@ -8,25 +8,29 @@
         "SignedOutCallbackPath ": "/signout-callback-oidc",
 
         // To call an API
-        "ClientSecret": "secret-goes-here"
+        "ClientSecret": "fvHwfoLA8Zls6Huj5MBUQUDrJ5HXrauFQKDd2CacxJQ="
+    },
+    "GraphBeta": {
```

Comment:
This is the case of a sample that calls several sevices:
- Graph
- And a downstream API (which turns out to be Graph v1.0, but could be something else.